shogun-lib/src/main/java/de/terrestris/shogun/lib/model/File.java

```diff
@@ -67,4 +67,8 @@
     @Column(length = Integer.MAX_VALUE)
     @Getter @Setter
     private byte[] file;
+
+    @Column
+    @Getter @Setter
+    private String path;
```

Comment:
Should be path of the file be ignored in JSON?

Reply:
I agree, this is an internal information and shouldn't be needed externally.